roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -6,7 +6,7 @@ dockerConfig:
   execHandlerName: ""
 imageConfig:
   format: {{ openshift.node.registry_url }}
-  latest: false
+  latest: true
```

Comment:
The format should be something like
'openshift3/ose-${component}:${version}' and the master interprets those
when asked for an image for a certain component. If latest is true then it
will always ask for the latest tag rather than the versioned tag.

On Wed, Aug 5, 2015 at 10:12 AM, Michal Minar <notifications@github.com>
wrote:

> In roles/openshift_node/templates/node.yaml.v1.j2
> <https://github.com/openshift/openshift-ansible/pull/419#discussion_r36305474>
> :
>
> > @@ -6,7 +6,7 @@ dockerConfig:
> >    execHandlerName: ""
> >  imageConfig:
> >    format: {{ openshift.node.registry_url }}
> > -  latest: false
> > +  latest: true
>
> I guess so. Let's put there some recent tag like v3.0.1.0.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/419/files#r36305474>.
>